docs/upgrading_to_project_factory_v10.0.md

```diff
@@ -0,0 +1,193 @@
+# Upgrading to Project Factory v10.0
+
+The v10.0 release of Project Factory is a backwards incompatible release for
+all modules since the breaking change is on
+[core_project_factory](../modules/core_project_factory) module which removes the
+need of gcloud and local-execs.
+
+## Migration Instructions
+
+1. Update variable default_service_account from down case to upper case if
```

Comment:
We should not force users to change this. There's no reason we can't handle it easily in the module for them: https://www.terraform.io/docs/configuration/functions/upper.html

Reply:
i know the upper function... since it's a breaking change I just thought a cleaner way would be better. i was wrong again :)